capture/capture_otbn.py

```diff
@@ -0,0 +1,764 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import random
+import signal
+import sys
+from dataclasses import dataclass
+from datetime import datetime
+from functools import partial
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+import yaml
+from lib.ot_communication import OTOTBNVERT, OTUART
+from project_library.project import ProjectConfig, SCAProject
+from scopes.cycle_converter import convert_num_cycles, convert_offset_cycles
+from scopes.scope import Scope, ScopeConfig, determine_sampling_rate
+from tqdm import tqdm
+
+sys.path.append("../")
+import util.helpers as helpers  # noqa: E402
+from target.cw_fpga import CWFPGA  # noqa: E402
+from util import check_version  # noqa: E402
+from util import plot  # noqa: E402
+from util import data_generator as dg  # noqa: E402
```

Comment:
```suggestion
import util.helpers as helpers
from target.cw_fpga import CWFPGA
from util import check_version
from util import data_generator as dg
from util import plot
```
Luckily we don't need the `sys.path.append("../")` anymore.